### PR TITLE
Make consumable via npm

### PR DESCRIPTION
**Pull request by mdarens**
- Make endpoint configurable

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "homepage": "https://github.com/civicsource/knockout-place-viewer#readme",
+  "dependencies": {
+    "jquery": ">=1.10"
+  },
+  "peerDependencies": {
+    "knockout": "3.x"
+  },
+  "devDependencies": {
+    "babel-eslint": "6.x",
+    "css-loader": "^0.23.1",
```
Comment (chadly) [inline package.json]: unused dep

**Comment by chadly** [comment]
You are going to have to update our usage of this with the new `endpoint` binding.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "knockout-place-viewer",
```
Comment (chadly) [inline package.json:2]: no mo dashes

Comment (mdarens) [inline package.json:2]: this one isn't in the webpack alias config. it's being required as `knockout-place-viewer`.

**Comment by mdarens** [comment]
The `endpoint` binding is optional and falls back to `window.civicsource.urls.fips` instead of expecting `app/payload` to resolve to something at runtime anywhere the module is used.

**Comment by chadly** [comment]
gotcha :shipit:

**Pull request merged**